lib/inputs.js

```diff
@@ -0,0 +1,17 @@
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function askForPassword(key) {
+  return new Promise((resolve) => {
+    readlineInterface.question(`Enter Password of ${key}:`, (password) => {
+      resolve(password);
+      readlineInterface.close();
+    });
+  });
+}
+
+exports.ask = askForPassword;
```

Comment:
I would recommend naming the export the same as what it exports

Reply:
Thx :) I have changed the export name